plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/product-image/index.tsx

```diff
@@ -32,7 +36,13 @@ const ProductImage = ( {
 				alt: '',
 		  };
 
-	return <img { ...imageProps } alt={ imageProps.alt } />;
+  return (
+    <img
+      { ...imageProps }
+      { ...( width && { width } ) }
+      { ...( height && { height } ) }
```

Comment:
```suggestion
      width={width}
      height={height}
```
Looking at the types for these props, it looks like they can be either a `number` or `undefined`. `undefined` attributes are getting ignored by react, so you don't have to do additional checks before setting them.